### DISCUSSION TITLE
Corrispettivi

### DISCUSSION THREAD

**Discussion by Picaland** [fattureincloud/api#204] [Q&A] [answered]
Salve a tutti, 
mi stavo chiedendo quale valore è più corretto assegnare a "type" nei parametri per la creazione di un Corrispettivo tramite API
https://developers.fattureincloud.it/api-reference/#post-/c/-company_id-/receipts

se till_receipt o sales_receipt

Grazie!

**Comment by emajo** [comment, marked as answer]
Ciao @Picaland, `till_receipt` è uno scontrino, mentre `sales_receipt` è una ricevuta.

Facci sapere se ti servono altre informazioni a riguardo,
grazie,
Emanuele.

**Reply by Picaland** [reply]
Ciao @emajo, grazie per la risposta e buon anno!
Avevo capiro la differenza in termine di significato ma mi interessava capire quale dei due parametri è più corretto usare se voglio creare dei corrispettivi. credo che sia **till_receipt** in quanto appunto si tratterebbe di scontrino fiscale digitale.

**Reply by emajo** [reply]
Ciao @Picaland, buon anno anche a te, purtroppo non sono la persona più adatta a rispondere a questo tipo di domanda, per avere delucidazioni di questo genere ti consiglio di scrivere al nostro supporto clienti che è molto più preparato su questi argomenti.

Grazie,
Emanuele.
